client/src/main.rs

```diff
@@ -724,6 +737,37 @@ fn add_cidr(interface: &InterfaceName, opts: &Opts, sub_opts: AddCidrOpts) -> Re
     Ok(())
 }
 
+fn rename_cidr(
+    interface: &InterfaceName,
+    opts: &Opts,
+    sub_opts: RenameCidrOpts,
+) -> Result<(), Error> {
+    let InterfaceConfig { server, .. } =
+        InterfaceConfig::from_interface(&opts.config_dir, interface)?;
+    let api = Api::new(&server);
+
+    log::info!("Fetching CIDRs");
+    let cidrs: Vec<Cidr> = api.http("GET", "/admin/cidrs")?;
+
+    if let Some((cidr_request, old_name)) = prompts::rename_cidr(&cidrs, &sub_opts)? {
+        log::info!("Renaming CIDR...");
+
+        let id = cidrs
+            .iter()
+            .filter(|c| c.name == old_name)
+            .map(|c| c.id)
+            .next()
+            .ok_or_else(|| anyhow!("CIDR not found."))?;
```

Comment:
I've tested this compiles:
```suggestion
        let id = cidrs
            .iter()
            .find(|c| c.name == old_name)
            .ok_or_else(|| anyhow!("CIDR not found."))?
            .id;
```